cogs/moderation.py

```diff
@@ -92,14 +92,14 @@ async def muterole(self, ctx: commands.Context):
         Set a role for mutes.
         """ 
         raise NotImplementedError('Command requires implementation and permission set-up.')
-
     
     @commands.hybrid_command()
-    async def blacklist(self, ctx: commands.Context):
+    async def ban(self, ctx: commands.Context, user: discord.User, *, reason: str):
         """
-        Ban a user that is not in the server.
+        Ban a user (including those that are not in the server).
```

Comment:
I would personally prefer if this command only worked for those not in the server, as I believe thats how it currently works in IB.ai. When possible, mods should use the built-in discord functionality to ban people, so this command should really only be used when a person has left the server, in my opinion.